src/components/NavBar/index.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import "./style.css";
+import { NavLink, useNavigate } from "react-router-dom";
+
+const NavBar = () => {
+  const history = useNavigate();
+
+  return (
+    <nav>
+      <NavLink
+        className={(navData) => (navData.isActive ? "active" : "none")}
```

Comment:
This is really cool, very concise method.
How is `navData` defined here though?

Reply:
navData is the parameter for the callback